src/styles/variables/colors.scss

```
@@ -11,289 +11,363 @@
-		var(--lightness-40) var(--brand-chroma-40) var(--primary-hue)
+		var(--lightness-40) var(--dynamic-chroma-40) var(--hue-primary)
 	);
-	--primary40_high: oklch(
-		var(--lightness-40) var(--brand-chroma-40) var(--primary-hue) /
-			var(--opacity-primary-high)
+	--primary40_semantic: oklch(
+		var(--lightness-40) var(--semantic-chroma-40) var(--hue-primary)
 	);
 	--primary40_medium: oklch(
-		var(--lightness-40) var(--brand-chroma-40) var(--primary-hue) /
+		var(--lightness-40) var(--dynamic-chroma-40) var(--hue-primary) /
```

Stylelint (17.14.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)